cmd/commands/cli/command.go

```diff
@@ -346,7 +346,7 @@ func (c *cliApp) connect(args []string) (err error) {
 }
 
 func (c *cliApp) mmnApiKey(args []string) (err error) {
-	profileUrl := c.config.GetStringByFlag(config.FlagMMNAddress) + "user/profile"
+	profileUrl := c.config.GetStringByFlag(config.FlagMMNAddress) + "/me"
```

Comment:
```suggestion
	profileUrl := strings.TrimSuffix(c.config.GetStringByFlag(config.FlagMMNAddress), "/") + "/me"
```

Would be nice to make sure we never put `//me` in the address